### PR TITLE
Cover uncovered modules, add debug for posting coverage, Resolves #127

### LINKED ISSUE #127: Missing coverage after updating from 0.8.0 to 0.9.1
After I upgraded to 0.9.1 (and blanket to 5e94fc30f2e694bb5c3718ddcbf60d467f4b4d26), when I see the coverage report from ember test, I am missing a great number of files, some of them seem to be files that do not have a test case. The report went from ~100 files to ~35 files. I added `excludeMissingFiles: false` to my blanket-options.

Ember version: 2.3.1

### PR DESCRIPTION
**Pull request by jschilli**
needs a refactor - debug etc and globals pollution kills the spirit.

new option for blanket-options.js

```
cliOptions: {
    debugCLI: true // emits additional diagnostic info
...
}
```

### CODE REVIEW

**Comment by edmundito** [comment]
I tested this locally for the issue I described in #112 and I can confirm that it is able to resolve the problem :+1:

File: vendor/start.js [hunk 1]
```diff
@@ -1,14 +1,29 @@
-/*global QUnit, blanket, mocha,  $ */
+/*global QUnit, blanket, mocha, moduleLoaderFinish, $ */
 
-function sendCoverage() {
+var debug = function (msg) {
+	var cliOptions = blanket.options('cliOptions') || {};
```
Comment (edmundito) [inline vendor/start.js]: Seems like you are missing an extra indentation space for this function.

**Comment by jschilli** [comment]
@edmundito thanks for the verification.

I've verified the uncovered issue and am awaiting further confirmation.

Will squash and repush once we've got a few more voices on this.

Any eyes on the README updates would also be helpful

File: vendor/blanket-require.js [hunk 2]
```diff
@@ -81,3 +92,27 @@ if (typeof exports !== 'undefined') {
         shouldExclude: shouldExclude
     };
 }
+
+
+/*
+ * After running all the tests we'll loop over all matching requirejs
+ * entries and annotate them so blanket will indicate their non-coverage
+ */
```
Comment (sglanzer) [inline vendor/blanket-require.js]: Thanks for pulling this back in, I hadn't noticed that it had been missed

File: README.md [hunk 3]
```diff
@@ -60,9 +60,14 @@ Append ```?coverage=true``` to the HTML report URL to enable coverage. This opti
 
 You can add this as part of the ```ember test``` command for a single run:
 
+*Pre ember-cli 1.13.13*
 ```bash
 ember test --test-page='tests/index.html?coverage=true'
 ```
+*Post ember-cli 1.13.13*
+```bash
+ember test --test_page=tests/index.html?coverage=true
```
Comment (sglanzer) [inline README.md]: Whew, that's a tight distinction - we should consider adding our own commands (`ember coverage`?) to insulate users from this

File: README.md [hunk 4]
```diff
@@ -114,6 +119,22 @@ As of release 0.2.4 loader exclusions are no longer necessary to fix conflicts w
 
 If you do not see any coverage reports at the bottom of the page, even after clicking the enable coverage checkbox, check your console for errors. If you see something like `TypeError: blanket.instrumentSync is not a function` you need to update your version of blanket. Currently, we are pointing to a fork of blanket, but if you run `ember g ember-cli-blanket` it will update your app to point to the correct blanket version.
 
+### No Coverage Reported
```
Comment (sglanzer) [inline README.md]: This is great, I'm thinking we should also add a TOC to the README to make these sections easier to find

**Comment by sglanzer** [comment]
Looks great @jschilli

**Pull request merged**